loader/checkpoint.go

```diff
@@ -95,36 +96,36 @@ func newRemoteCheckPoint(tctx *tcontext.Context, cfg *config.SubTaskConfig, id s
 		finishedTables: make(map[string]struct{}),
 		schema:         cfg.MetaSchema,
 		table:          fmt.Sprintf("%s_loader_checkpoint", cfg.Name),
-		tctx:           newtctx,
+		tctx:           tctx.WithLogger(tctx.L().WithFields(zap.String("component", "remote checkpoint"))),
 	}
 
-	err = cp.prepare()
+	err = cp.prepare(tctx2)
```

Comment:
is `defaultDBContextTimeout(10 seconds)` enough?

Reply:
sometimes tidb is not fast, espicallly DDL

Reply:
In fact, I'm not sure, but this timeout only used for some checkpoint operations and they may not expensive (and if timeout, they can be retried through restring the task). Are there any suggestions for me?

BTW, "better improvement for these operations can be done when moving them to etcd"  in the PR's description, and it's better to add `context` support for all I/O (network) operations later, then we can control the timeout in the upper caller.